common/docker-entrypoint.d/00-check-for-required-env.sh

```diff
@@ -49,7 +49,7 @@ elif [[ -v AWS_WEB_IDENTITY_TOKEN_FILE ]]; then
 # If none of the options above is used, require static credentials.
 # See https://docs.aws.amazon.com/sdkref/latest/guide/feature-static-credentials.html.
 else
-  required+=("S3_ACCESS_KEY_ID" "S3_SECRET_KEY")
+  required+=("S3_ACCESS_KEY_ID" "S3_SECRET_KEY" "S3_SESSION_TOKEN")
```

Comment:
I'm a bit confused about this line. What about the state of the else condition makes the presence of `S3_SESSION_TOKEN`?

Excuse my original comment about this file because I may have been unclear. I was thinking that we would just do something like is at the [bottom of the file](https://github.com/nginxinc/nginx-s3-gateway/blob/master/common/docker-entrypoint.d/00-check-for-required-env.sh#L111).

For example:
```bash
if [[ ! -v "S3_SESSION_TOKEN" ]]; then
  echo "S3 Session token present"
fi
```
(Please check my above conditional syntax for it may be wrong)

Additional, I ran the gateway without `S3_SESSION_TOKEN` (only with `S3_ACCESS_KEY` and `S3_SECRET_KEY`) and it seemed to work fine without triggering a failure state. I'm scratching my head wondering why...

Reply:
Thanks for finding this.